google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/CloseStream.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.models;
+
+import com.google.api.core.InternalApi;
+import com.google.bigtable.v2.ReadChangeStreamResponse;
+import com.google.bigtable.v2.StreamContinuationToken;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.List;
+import javax.annotation.Nonnull;
+
+public final class CloseStream implements ChangeStreamRecord, Serializable {
+  private static final long serialVersionUID = 7316215828353608505L;
+  private final com.google.rpc.Status status;
+  private transient ImmutableList.Builder<ChangeStreamContinuationToken>
+      changeStreamContinuationTokens = ImmutableList.builder();
+
+  private CloseStream(
+      com.google.rpc.Status status, List<StreamContinuationToken> continuationTokens) {
```

Comment:
I noticed there's also a StreamContuniationTokens class, where is that used? Is List<StreamContinuationToken> basically StreamContinuationToken**s**?

Reply:
Why specifying the path to Status instead of just importing com.google.rpc.Status?